src/components/card/Card.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+import * as React from 'react';
+import classNames from 'classnames';
+
+import './Card.scss';
+
+type Props = {
+    children: React.Node,
+    className?: string,
+    hasPadding?: boolean,
+    title?: React.Node,
+};
+
+const Card = ({ children, className = '', hasPadding = true, title, ...rest }: Props) => (
+    <div
+        className={classNames(`bdl-Card ${className}`, {
+            'bdl-Card--padded': hasPadding,
```

Comment:
CSS classes ideally should avoid including specific visual aspects in their names.